config/routes.rb

```diff
@@ -397,5 +397,7 @@ def redirect(path, options = { prefix: Whitehall.router_prefix })
 
   mount SidekiqGdsSsoMiddleware, at: "/sidekiq"
 
+  mount Flipflop::Engine => "/flipflop"
```

Comment:
I was sort of expecting to see the development environment constraint defined here, as is done for the `GovukPublishingComponents::Engine` in the line below. If it doesn't functionally change anything, it might be nicer to keep this logic in the same place rather than defining it in `application.rb`.

e.g. this

```suggestion
  mount Flipflop::Engine => "/flipflop" if Rails.env.development?
```

and then remove the `config.flipflop.dashboard_access_filter` before filter from `application.rb`.

Reply:
Good shout

Reply:
Actually this isn't going to work because that config option is required by the FlipFlop engine. I could return true from it and move the environment check into the routes file but that doesn't seem a massive improvement.

Reply:
That seems fair. My comment definitely wasn't a _strong_ opinion, so all good.